pkg/network/infraconfigurators/bridge.go

```diff
@@ -179,10 +179,6 @@ func (b *BridgePodNetworkConfigurator) GenerateDomainIfaceSpec() api.Interface {
 	return api.Interface{
 		MAC: &api.MAC{MAC: b.vmMac.String()},
 		MTU: &api.MTU{Size: strconv.Itoa(b.podNicLink.Attrs().MTU)},
```

Comment:
MAC and MTU can be taken at phase2 too from the pod interface in case of no IPAM and from "-nic" interface in case of IPAM for the bridge binding

Reply:
Do you know why changing podnic MAC is needed ?